src/pages/overlay/components/AmbassadorList.tsx

```diff
@@ -11,10 +11,29 @@ import AnimalButton from '../../../utils/global/animalButton/AnimalButton'
 import styles from './ambassadorList.module.css'
 import arrow from '../../../assets/arrow.jpg'
 
+const SCROLL_OFFSET = 200;
+
 export interface AmbassadorListProps{
     showAnimalList: boolean
     chatChosenAmbassador?: string
 }
+
+function scrollListToAnimal(listElement: HTMLDivElement | null, name: string) {
```

Comment:
Move function to the inside of AmbassadorList function. Move to under the useEffect

Reply:
should be fixed